src/pages/details/components/tag/tagLink.tsx

```diff
@@ -77,19 +77,19 @@ class TagLinkBase extends React.Component<TagLinkProps> {
     const { filterBy, groupBy, id, report, reportPathsType } = this.props;
     const { isOpen } = this.state;
 
-    const tagKeyValues = [];
+    let tagKeyValuesCount = 0;
 
     if (report) {
       for (const tag of report.data) {
-        for (const val of tag.values) {
-          tagKeyValues.push(`${(tag as any).key}: ${val}`);
+        for (const {} of tag.values) {
```

Comment:
why not doing `tagKeyValuesCount += tag.values.length()`

Reply:
Yes, that would be better. This was old code and didn't see that. Thanks!

Reply:
Yes, that would be better. This was an old code block, previously used to concatenate strings. Thanks!